src/watertap_contrib/seto/property_models/basic_water_properties.py

```diff
@@ -0,0 +1,401 @@
+"""
+This module contains a basic property package for simple water treatment models.
```

Comment:
Before I do anything here, can you update me on your plans for migration directly to WaterTAP, @kurbansitterley? I think I know how to fix the errors cropping up, but don't want to start commits if this will all be abandoned and put up as a separate PR on WaterTAP.

Reply:
The plan is to essentially merge the EC ZO version on the WaterTAP branch with this model. However, that model is pretty much one constraint atm so it will be mostly this model. I have converted this model to use the ZO framework on the [ec_zo_model](https://github.com/watertap-org/watertap/tree/ec_zo_model) branch of WaterTAP, but it isn't quite in PR shape yet (but will be soon).